docs/proposals/keadm-scope.md

```diff
@@ -134,32 +134,30 @@ In cloud node it shuts down the cloud processes of KubeEdge
 In edge node it shuts down the edge processes of KubeEdge
 
 Usage:
-keadm reset [flags]
+  keadm reset [flags]
 
 Examples:
 
 For cloud node:
 keadm reset
 
 For edge node:
-keadm reset --k8sserverip 10.20.30.40:8080
+keadm reset
 
 
 Flags:
+      --force                Reset the node without prompting for confirmation
   -h, --help                 help for reset
-  -k, --k8sserverip string   IP:Port address of cloud components host/VM
+      --kube-config string   Use this key to set kube-config path, eg: $HOME/.kube/config (default "/root/.kube/config")
 
 ```
 
 ### keadm join --help
 
 ```
 
-"keadm join" command bootstraps KubeEdge's edge component.
-It checks if the pre-requisites are installed already,
```

Comment:
why remove these lines?

Reply:
<img width="699" alt="image" src="https://user-images.githubusercontent.com/76980726/170233507-7fe55cf9-5459-44dc-b4c8-3b8329502524.png">
This is the output for command `keadm join --help`，the output for code has been changed.

Reply:
ok